modules/ztp-configuring-ipsec-using-ztp-and-siteconfig.adoc

```diff
@@ -44,23 +41,21 @@ interfaces:
 - name: hosta_conn
   type: ipsec
   libreswan:
-    left: <cluster_node> <1>
+    left: '%defaultroute'
     leftid: '%fromcert'
     leftmodecfgclient: false
-    leftcert: <left_cert> <2>
+    leftcert: left_server
     leftrsasigkey: '%cert'
-    right: <external_host> <3>
+    right: <external_host> <1>
     rightid: '%fromcert'
     rightrsasigkey: '%cert'
-    rightsubnet: <external_address> <4>
-    ikev2: insist <5>
+    rightsubnet: <external_address> <2>
+    ikev2: insist <3>
     type: tunnel
 ----
-<1> Replace `<cluster_node>` with the IP address or DNS hostname of the cluster node for the cluster-side IPsec tunnel.
-<2> Replace `<left_cert>` with the IPsec certificate nickname.
```

Comment:
This is still valid and should remain.

Reply:
These updates were made based on the following suggestions from @sabinaaledort 

- https://github.com/openshift/openshift-docs/pull/80997#discussion_r1764913489
- https://github.com/openshift/openshift-docs/pull/80997#discussion_r1764907052


Could you please confirm if these updates are fine? @bradyallenjohnson

Reply:
Ok, as long as you suggest the left_cert field should match, its fine.